.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,56 @@
+name: 'Bug report'
+labels: ['bug', 'triage']
+description: Create a report to help Epinio to improve
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. In this environment...
+      2. With this config...
+      3. Run '...'
+      4. See error...
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      examples:
+        - **OS**: Linux
+        - **Architecture**: arm64
+    value: |
+        - OS:
+        - Architecture:
+    render: markdown
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Anything else?
+    description: |
+      Links? References? Screenshots? Anything that will give us more context about the issue you are encountering!
```

Comment:
This is also very generic. We care more about kubectl logs than about screenshots I think.

Reply:
Based on your suggestion, I added a log section based on the one from epinio/epinio